openassessment/xblock/xml.py

```diff
@@ -606,9 +610,21 @@ def serialize_training_examples(examples, assessment_el):
 
         # Answer provided in the example (default to empty string)
         answer_el = etree.SubElement(example_el, 'answer')
-        for part in example_dict.get('answer', {}).get('parts', []):
-            part_el = etree.SubElement(answer_el, 'part')
-            part_el.text = unicode(part.get('text', u''))
+        try:
+            answer = example_dict.get('answer')
```

Comment:
@iloveagent57 We are initiating `parts` var here, Why we are not using it anywhere?